src/__tests__/ColorDot.test.tsx

```diff
@@ -0,0 +1,15 @@
+// __tests__/ColorDot.test.ts
+import React from "react";
+import { render, screen } from "@testing-library/react";
+import "@testing-library/jest-dom";
+import { ColorDot } from "..";
+
+describe("ColorDot", () => {
+  test("renders at specified size", () => {
+    render(<ColorDot size={5} color={"red"} />);
+
+    const element = screen.getByRole("color-dot");
+
+    expect(element.style.width && element.style.height).toBe("5px");
```

Comment:
(item1 && item2 ) will result in only checking item2, to check both duplicate the expectation or map to a config
this is valid for use cases where you want to check another value exists before checking item2